### PR TITLE
オペレータ権限設定

### PR DESCRIPTION
**Pull request by TimLai125**
@yostashiro I plan to test the module a few more times. Please kindly advice some concerned patterns since I am not very familiar with the access right customization.

Furthermore, we need to update the SST environment since there is an issue related to the hr module.
Related Github issue: https://github.com/odoo/odoo/pull/22686
Merged commit: https://github.com/odoo/odoo/commit/5a6123c699a1406191a00c3c9a09e6f320a43867

### CODE REVIEW

File: model_security_adjust_sst/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Model Security Adjust SST',
+    'version': '11.0.1.1.0',
+    'author': 'Quartile Limited',
+    'website': 'https://www.odoo-asia.com',
+    'category': 'Purchase',
```
Comment (yostashiro) [inline model_security_adjust_sst/__manifest__.py]: Let's say "Security"

File: model_security_adjust_sst/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Model Security Adjust SST',
+    'version': '11.0.1.1.0',
```
Comment (yostashiro) [inline model_security_adjust_sst/__manifest__.py]: 11.0.1.0.0

File: model_security_adjust_sst/security/hr_security.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    <record id="group_hr_operator" model="res.groups">
+        <field name="name">Operator</field>
+        <field name="category_id" ref="base.module_category_human_resources"/>
+        <field name="implied_ids" eval="[(4, ref('base.group_user'))]"/>
+    </record>
+
+    <record id="hr.group_hr_user" model="res.groups">
+        <field name="implied_ids"
+               eval="[(3, ref('base.group_user')), (4, ref('group_hr_operator'))]"/>
```
Comment (yostashiro) [inline model_security_adjust_sst/security/hr_security.xml]: Doesn't this work?
```.xml
<field name="implied_ids" eval="[(6, ref('group_hr_operator'))]"/>
```

File: model_security_adjust_sst/security/hr_security.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+    <record id="operator_hr_employee_perm" model="ir.rule">
+        <field name="name">operator: hr.employee permission</field>
+        <field name="model_id" ref="hr.model_hr_employee"/>
+        <field name="groups" eval="[(4, ref('group_hr_operator'))]"/>
+        <field name="perm_read" eval="True"/>
+        <field name="perm_write" eval="True"/>
+        <field name="perm_create" eval="False"/>
+        <field name="perm_unlink" eval="False"/>
+    </record>
```
Comment (yostashiro) [inline model_security_adjust_sst/security/hr_security.xml]: This record is not needed.  The record in ir.model.access.csv does the job.

File: model_security_adjust_sst/security/stock_security.xml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+    <record id="group_stock_operator" model="res.groups">
+        <field name="name">Operator</field>
+        <field name="category_id" ref="base.module_category_warehouse_management"/>
+        <field name="implied_ids" eval="[(4, ref('stock.group_stock_user'))]"/>
+    </record>
+
+    <record id="stock.group_stock_manager" model="res.groups">
+        <field name="implied_ids"
+               eval="[(3, ref('stock.group_stock_user')), (4, ref('group_stock_operator'))]"/>
```
Comment (yostashiro) [inline model_security_adjust_sst/security/stock_security.xml]: Doesn't this work?
```.xml
<field name="implied_ids" eval="[(6, ref('group_stock_operator'))]"/>
```

File: model_security_adjust_sst/security/stock_security.xml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+    <record id="operator_stock_move_perm" model="ir.rule">
+        <field name="name">operator: stock.move permission</field>
+        <field name="model_id" ref="stock.model_stock_move"/>
+        <field name="groups" eval="[(4, ref('group_stock_operator'))]"/>
+        <field name="perm_read" eval="True"/>
+        <field name="perm_write" eval="True"/>
+        <field name="perm_create" eval="True"/>
+        <field name="perm_unlink" eval="True"/>
+    </record>
```
Comment (yostashiro) [inline model_security_adjust_sst/security/stock_security.xml]: This record is not needed. The record in ir.model.access.csv does the job.

File: model_security_adjust_sst/views/stock_menu_views.xml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.menu" id="stock.menu_warehouse_report">
+        <field name="groups_id"
+               eval="[(3, ref('stock.group_stock_manager')), (4, ref('group_stock_operator'))]"/>
```
Comment (yostashiro) [inline model_security_adjust_sst/views/stock_menu_views.xml]: Doesn't this work?
```.xml
<field name="groups_id" eval="[(6, ref('group_stock_operator'))]"/>
```

File: model_security_adjust_sst/__manifest__.py (hunk 1 above)
Comment (TimLai125) [reply model_security_adjust_sst/__manifest__.py]: Ok, noted. I will fix it

File: model_security_adjust_sst/__manifest__.py (hunk 2 above)
Comment (TimLai125) [reply model_security_adjust_sst/__manifest__.py]: Ok, noted. I will fix it

File: model_security_adjust_sst/security/hr_security.xml (hunk 3 above)
Comment (TimLai125) [reply model_security_adjust_sst/security/hr_security.xml]: Yes, it is right. We could have this for groups

File: model_security_adjust_sst/security/stock_security.xml (hunk 5 above)
Comment (TimLai125) [reply model_security_adjust_sst/security/stock_security.xml]: Yes, it is right. We could have this for groups

File: model_security_adjust_sst/security/hr_security.xml (hunk 4 above)
Comment (TimLai125) [reply model_security_adjust_sst/security/hr_security.xml]: This part is deleted

File: model_security_adjust_sst/security/stock_security.xml (hunk 6 above)
Comment (TimLai125) [reply model_security_adjust_sst/security/stock_security.xml]: This part is deleted

File: model_security_adjust_sst/views/stock_menu_views.xml (hunk 7 above)
Comment (TimLai125) [reply model_security_adjust_sst/views/stock_menu_views.xml]: Yes it does, I have fixed this

**Pull request merged**